src/Tizen.Multimedia.Camera/Camera/Camera.cs

```diff
@@ -58,56 +58,46 @@ public partial class Camera : IDisposable, IDisplayable<CameraError>
         /// <feature> http://tizen.org/feature/camera </feature>
         public Camera(CameraDevice device)
         {
```

Comment:
How about checking validity here?
```suggestion
        {ValidationUtil.ValidateEnum(typeof(CameraDevice), device, nameof(device));}
```

Reply:
OK I'll add it.